### DISCUSSION TITLE
İbax Cüzdan Güncellemesi

### DISCUSSION THREAD

**Discussion by Majezikonline** [IBAX-io/go-ibax#462] [Ideas] [answered]
### Clear and concise description of the problem

İBAX coinini uzun süre 6-12 ay elinde tutup satmayan arkadaşlara özel gönderim ücretlerinin getirilmesi sağlanmalı. 

### Suggested solution

6-12 ay arası İBAX coin'i hodl yapanlara özel; İBAX cüzdanı ile farklı cüzdanlara ibax coin gönderirken, gönderim ücretlerinin en düşük seviyeye çekilmesi gerekli.

### Alternative

Forumda aktif ve ibax hodl eden üyelerin gönderim ücretinde mağduriyet yaşamaması adına gönderim ücreti kadar İBAX coinin cüzdanlara aktarılması.

### Validations

- [X] Read the [docs](https://docs.ibax.io).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by Emircangzc** [comment]
The write is true

**Comment by Astalmeta7** [comment]
I think it's very reasonable. After all, the common thought of all of us is that this project will come to a very good place, isn't it?

**Comment by Majezikonline** [comment]
> Bence çok mantıklı. Sonuçta hepimizin ortak düşüncesi bu projenin çok iyi yerlere geleceği değil mi?

Umarım ♥️

**Comment by Majezikonline** [comment]
> yazılan doğrudur

♥️

**Comment by ibaxio** [comment, marked as answer]
According to the rules of the IBAX network, after the official network is online, the NFT mining will be activated, and the IBXC stake to the IBAX Network, thereby obtaining minging rewards.

It is recommended to keep the IBXC you redeemed.
